docs/_includes/cmake_in_clickhouse_footer.md

```diff
@@ -1,121 +0,0 @@
-
```

Comment:
The information in this file is overall valid and can't go out-of-sync easily. On the other hand, the guidelines are obvious and there is not a big benefit of keeping them around. I am fine with removing this file.